test/test-network/conf/25-dummy.link

```diff
@@ -0,0 +1,5 @@
+# SPDX-License-Identifier: LGPL-2.1-or-later
```

Comment:
Why .link is necessary for the test? Please drop.

Reply:
Done.